kivy/graphics/instructions.pyx

```diff
@@ -654,6 +654,11 @@ cdef class Canvas(CanvasBase):
         return 0
 
     cpdef add(self, Instruction c):
+        '''Add a new :class:`Instruction` to our list. If the list contains
+        the after group, it remains the last one. This is different to how
+        :meth:`insert` works, which doesn't care the before/after group at
```

Comment:
```suggestion
        :meth:`insert` works, which doesn't care about the before/after group at
```
i think this is clear enough, but if you want, you could try something like
```
"""
Append an :class:`Instruction` to our list. If the canvas contains an `after`
group, then this instruction is inserted just before the after group, which
remains last. This is different from how :meth:`insert works, which can
insert anywhere.
"""
```

Reply:
I really appriciate :+1: I'll take your one.

Reply:
@tshirtman I'm ok with the current state of the pr.